plugins/services/src/js/reducers/serviceForm/JSONReducers/Containers.js

```diff
@@ -287,6 +295,17 @@ function containersParser(state) {
   }, []);
 }
 
+function deleteContainerImageCheck(image) {
```

Comment:
Renamed and yes (image == null || !image.id) now makes it work exactly like the single container